graylog2-server/src/main/java/org/graylog2/rest/resources/system/inputs/ExtractorsResource.java

```diff
@@ -120,14 +116,7 @@ public Response create(@ApiParam(name = "inputId", required = true)
                            @Valid @NotNull CreateExtractorRequest cer) throws NotFoundException {
         checkPermission(RestPermissions.INPUTS_EDIT, inputId);
 
-        final MessageInput input = inputs.getRunningInput(inputId);
-        if (input == null) {
-            final String msg = "Input <" + inputId + "> not found.";
-            LOG.error(msg);
-            throw new javax.ws.rs.NotFoundException(msg);
-        }
-
-        final Input mongoInput = inputService.find(input.getPersistId());
+        final Input mongoInput = inputService.find(inputId);
```

Comment:
I think we should test for `null` here. If we don't it's probably harmless, but the error wouldn't be nice.